src/jit/morph.cpp

```diff
@@ -7351,6 +7351,10 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
  */
 void Compiler::fgMorphTailCall(GenTreeCall* call, void* pfnCopyArgs)
 {
+#if defined(_TARGET_UNIX_)
+    noway_assert("Slow tail calls not supported on non-windows platforms.");
```

Comment:
This will never fail; you need a preceding `!`.

And please capitalize `Windows`.

Reply:
Thanks changing